src/EVault/modules/Liquidation.sol

```diff
@@ -125,9 +124,13 @@ abstract contract LiquidationModule is ILiquidation, Base, BalanceUtils, Liquidi
         // Compute discount
 
         uint256 discountFactor = liquidityCollateralValue * 1e18 / liquidityLiabilityValue; // discountFactor = health score = 1 - discount
-
-        if (discountFactor < 1e18 - MAXIMUM_LIQUIDATION_DISCOUNT) {
-            discountFactor = 1e18 - MAXIMUM_LIQUIDATION_DISCOUNT;
+        {
+            uint256 minDiscountFactor;
+            unchecked {
+                // discount <= config scale, so discount factor >= 0
+                minDiscountFactor = 1e18 - uint256(1e18) * vaultStorage.maxLiquidationDiscount.toUint16() / CONFIG_SCALE;
```

Comment:
The `maxLiquidationDiscount` configuration on the `EVault` deployment is not enforced. This means that a default `maxLiquidationDiscount = 0` is preconfigured. 

When `maxLiquidationDiscount == 0` the `minDiscountFactor = 1e18 - 0 = 1e18` → `discountFactor = 1e18` → `maxYieldValue = maxRepayValue * 1e18 / 1e18 = maxRepayValue`

This means that the liquidator won't get any bonus for repaying the violator's debt.

Should the `Governor` be forced to configure deliberately an initial `maxLiquidationDiscount` when the `EVault` is deployed?  Or is this a valid scenario for you?